system/oils/doinst.sh

```diff
@@ -0,0 +1,12 @@
+if [ ! -r etc/shells ]; then
+  touch etc/shells
+  chmod 644 etc/shells
+fi
+
```

Comment:
If we are going to add these automatically in doinst.sh, should we remove them automatically in douninst.sh?